ui/raidboss/data/04-sb/alliance/royal_city_of_rabanastre.txt

```diff
@@ -41,17 +41,17 @@ hideall "--start--"
 1078.0 "Earth Hammer" duration 13.5 # start of cast -> damage
 1100.0 "Quake IV" # end of cast
 1109.0 "Rock Cutter" # end of cast
-1112.5 "--sync--" sync /:Command Tower starts using Command Tower/ window 100,100
+1112.5 "--sync--" sync /:266E:command tower/ window 100,100
```

Comment:
Is this lowercase?

Reply:
Since i play in german i cant really verify this one.
But my script returned the lower case version (bnpcname > ID:6950)

Reply:
As with everything coming out of the FFXIV plugin, it is in proper case.  All regexes are case insensitive anyway, I just didn't understand why this needed a change.